components/cdc/src/delegate.rs

```diff
@@ -393,7 +390,6 @@ impl Delegate {
         let mut change_data_event = Event::default();
         change_data_event.region_id = self.region_id;
         change_data_event.event = Some(Event_oneof_event::ResolvedTs(resolved_ts.into_inner()));
-        self.broadcast(change_data_event, 0, true);
```

Comment:
What if there are multiple TiCDC observe the same region?

Reply:
these are some redundant code. 🤣